moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -1132,7 +1136,7 @@ bool MoveItConfigData::inputROSControllersYAML(const std::string& file_path)
   std::ifstream input_stream(file_path.c_str());
   if (!input_stream.good())
   {
-    ROS_ERROR_STREAM_NAMED("ros_controller.yaml", "Unable to open file for reading " << file_path);
+    ROS_INFO_STREAM_NAMED("ros_controller.yaml", "Does not exist " << file_path);
```

Comment:
shouldn't this be an error or warning?

Reply:
I agree a `warning` maybe better here, initially the package might not has a `ros_controllers.yaml`, so a `warn` would do I believe.